testutils/src/main/java/com/hyperwallet/android/ui/testutils/util/DateConversion.java

```diff
@@ -0,0 +1,22 @@
+package com.hyperwallet.android.ui.testutils.util;
+
+
+import static android.text.format.DateUtils.formatDateTime;
+
+import android.content.Context;
+
+import com.hyperwallet.android.ui.common.util.DateUtils;
+
+import java.util.Date;
+
+public class DateConversion {
+
+    public static String convertDate(Context context, String dateString, int flag) {
+        Date date = DateUtils.fromDateTimeString(dateString);
+        return convertDate(context, date, flag);
```

Comment:
See if ApplicationProvider.getApplicationContext() works instead of passing in context as a parameter.  I think we can also move the flags into this class and expose two methods. One for returning the date found in the receipt details and another for returning the date found in the receipt list.